packages/themes/dekode-theme/src/style.css

```diff
@@ -32,18 +27,19 @@ img {
 	}
 }
 
-.entry-header {
+.site-container {
```

Comment:
Could this helper class be moved to T2 @walbo? We usually need this on all projects, especially in header and footer.